geopandas/base.py

```diff
@@ -73,8 +80,8 @@ def _delegate_property(op, this):
         return Series(data, index=this.index)
 
 
-def _delegate_geo_method(op, this, *args, **kwargs):
-    # type: (str, GeoSeries) -> GeoSeries
+def _delegate_geo_method(op: str, this: "GeoSeries", *args, **kwargs) -> "GeoSeries":
+    # type: (str, GeoSeries) -> "GeoSeries"
```

Comment:
I thought if there are annotations, types in docs could be deleted, beacause duplicated.

Reply:
Just to be clear: not in actual user-facing docstrings, only in such comments as the above